index.js

```diff
@@ -21,7 +22,7 @@ const validateJson = async (filePath) => {
     try {
         JSON.parse(await fs.readFile(filePath, 'utf8'));
     } catch (error) {
-        console.error('Failed to parse services.json', error);
+        console.error(chalk.red.bold('Failed to parse services.json: ') + chalk.red(error.message));
```

Comment:
is it ok for highlight only heading — `Failed to parse services.json` — and the rest to log by console.log()?

Reply:
in the case of `services.json` - looks normal, but with other errors - will visually blend

<details><summary>services.json error:</summary>

<img width="715" alt="Screenshot 2023-10-17 at 1 54 06 PM" src="https://github.com/AdguardTeam/HostlistsRegistry/assets/91964807/a38cac16-9716-4d5a-823f-2ba60ea5a4b4">

</details><br/>

<details><summary>svg error, removed services:</summary>

<img width="865" alt="Screenshot 2023-10-17 at 1 50 08 PM" src="https://github.com/AdguardTeam/HostlistsRegistry/assets/91964807/45202de9-5a6b-4ca2-ae36-4c9404a17bef">

</details><br/>

Reply:
When error/warning message in in a new line(s), better add indentation to improve readability 
![image](https://github.com/AdguardTeam/HostlistsRegistry/assets/8361299/e9c0935c-4b23-47b8-98ff-305bc9ff1f28)